src/lerobot/policies/act/modeling_act.py

```diff
@@ -146,8 +150,11 @@ def predict_action_chunk(self, batch: dict[str, Tensor]) -> Tensor:
         actions = self.unnormalize_outputs({ACTION: actions})[ACTION]
         return actions
 
-    def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, dict]:
+    def forward(self, batch: dict[str, Tensor], **kwargs: object) -> tuple[Tensor, dict]:
```

Comment:
`**kwargs` should be typed using `typing.TypedDict` and `typing_extensions.Unpack`.